weather_dl/download_pipeline/parsers.py

```diff
@@ -299,15 +303,15 @@ def date_range(start: datetime.date, end: datetime.date, increment: int = 1) ->
     return (start + datetime.timedelta(days=x) for x in range(0, (end - start).days + offset, increment))
 
 
-def _parse_lists(config_parser: configparser.ConfigParser, section: str = '') -> t.Dict:
-    """Parses multiline blocks in *.cfg files as lists."""
-    config = dict(config_parser.items(section))
-
+def _parse_lists(config: dict, section: str = '') -> t.Dict:
+    """Parses multiline blocks in *.cfg and *.json files as lists."""
     for key, val in config.items():
-        if '/' in val and 'parameters' not in section:
-            config[key] = parse_mars_syntax(val)
-        elif '\n' in val:
-            config[key] = _splitlines(val)
+        # Checks str type for backward compatibility since it also support "padding": 0 in json config
+        if isinstance(val, str):
```

Comment:
style nit: consider "returning" early (or, `continue`ing early) instead of deeper nesting.